taichi/system/timer.cpp

```diff
@@ -69,6 +69,27 @@ void Time::sleep(double s) {
   Time::usleep(s * 1e6_f64);
 }
 
+void Time::wait_until(double t) {
+#ifndef _WIN64  // we can't release CPU slice due to the lack of
+                // microsecond (us) sleep on Windows... sadly.
+  double dt;
+  if (t < Time::get_time()) {
+    return;
+  }
+  do {  // use system-provided sleep for large scale sleeping:
+    dt = t - Time::get_time();
+    if (dt <= 0) {
+      return;
+    }
+    Time::sleep(dt * (dt < 4e-2_f64 ? 0.02 : 0.4));
```

Comment:
Is this condition any different here ( using 0.02 or 0.4 )?
Why not just use :
```suggestion
    Time::sleep(dt * 0.4);
```

Reply:
This ensures that the dt of sleep is more accurate when small; without this, I found the FPS to be around 59; with this, I found the FPS to be accurately 60.00.